src/lib/datepicker/calendar.html

```diff
@@ -24,18 +24,17 @@
   <mat-month-view
       *ngSwitchCase="'month'"
       [(activeDate)]="_activeDate"
-      [selected]="selected"
+      [selectionModel]="selectionModel"
       [dateFilter]="dateFilter"
       [maxDate]="maxDate"
       [minDate]="minDate"
-      (selectedChange)="_dateSelected($event)"
       (_userSelection)="_userSelected()">
   </mat-month-view>
 
   <mat-year-view
       *ngSwitchCase="'year'"
       [activeDate]="_activeDate"
-      [selected]="selected"
+      [selected]="selectionModel.selected"
```

Comment:
We should probably pass the full selection model. For example say I select a start date somewhere in February and then switch to month view. When I hove over March I would want it to show a preview range from February-March